TypeError transforming bnf/salt/data/salt-334.xml

```
docker run --rm -e SKIP_FETCH_CONFIG=true                 -e SKIP_FETCH_DATA=true                 -v $(pwd)/.:/opt/traject                 -v $(pwd)/../dlme-traject:/opt/traject/config                 -v $(pwd)/../dlme-metadata:/opt/traject/data                 -v $(pwd)/output:/opt/traject/output                 suldlss/dlme-transform:latest bnf/salt/data/salt-334.xml
```

results in
```
2019-03-01T20:06:10+00:00  INFO    Traject::Indexer with 1 processing threads, reader: TrajectPlus::XmlReader and writer: DlmeJsonResourceWriter
2019-03-01T20:06:10+00:00 ERROR Unexpected error on record <record #1 (data/bnf/salt/data/salt-334.xml #1), output_id:PFRBI874,http://www.archives.saltresearch.org/R/?func=dbin-jump-full&object_id=827168>
    while executing (to_field "cho_edm_type" at config/bnf_salt_config.rb:9)

    Record: <?xml version="1.0"?>
<srw:record xmlns:srw="http://www.loc.gov/zing/srw/" xmlns:ns7="http://gallica.bnf.fr/namespaces/gallica/" xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:onix_dc="http://bibnum.bnf.fr/NS/onix_dc/" xmlns:onix="http://www.editeur.org/onix/2.1/reference/" xmlns:dc="http://purl.org/dc/elements/1.1/">
            <srw:recordSchema>http://www.openarchives.org/OAI/2.0/OAIdc.xsd</srw:recordSchema>
            <srw:recordPacking>xml</srw:recordPacking>
            <srw:recordData>
                <oai_dc:dc>
                    <dc:contributor>Editor: Ernest Leroux</dc:contributor>
                    <dc:coverage>Paris</dc:coverage>
                    <dc:date>1874-12-15</dc:date>
                    <dc:description>Recueil mensuel publi&#xE9; par la Librairie Ernest Leroux. Osmanl&#x131; tarihine ili&#x15F;kin bibliyografya i&#xE7;erir - A bibliography about Ottoman History is included.</dc:description>
                    <dc:format>16-23 cm</dc:format>
                    <dc:identifier>PFRBI874</dc:identifier>
                    <dc:identifier>http://www.archives.saltresearch.org/R/?func=dbin-jump-full&amp;object_id=827168</dc:identifier>
                    <dc:language>French</dc:language>
                    <dc:publisher>Imp. Eugene Heutte et cie</dc:publisher>
                    <dc:relation>vignette : http://www.archives.saltresearch.org/webclient/DeliveryManager?pid=856748</dc:relation>
                    <dc:rights>Open access</dc:rights>
                    <dc:title>Revue Bibliographique de Philologie et d'Histoire</dc:title>
                </oai_dc:dc>
            </srw:recordData>
            <srw:recordIdentifier>ark</srw:recordIdentifier>
            <srw:recordPosition>33</srw:recordPosition>
            <srw:extraRecordData>
                <epubFile/>
                <infoSupModifiable/>
                <link>http://www.archives.saltresearch.org/R/?func=dbin-jump-full&amp;object_id=827168 </link>
                <nqamoyen>0.0</nqamoyen>
                <thumbnail> http://www.archives.saltresearch.org/webclient/DeliveryManager?pid=856748 </thumbnail>
                <typedoc>periodiques</typedoc>
            </srw:extraRecordData>
        </srw:record>

    Exception: TypeError: no implicit conversion of String into Array
    /usr/local/bundle/gems/traject_plus-1.0.0/lib/traject_plus/macros.rb:56:in `replace'

[ERROR] no implicit conversion of String into Array
```

--- a/bnf_common_config.rb
+++ b/bnf_common_config.rb
@@ -16,21 +16,21 @@ settings do
 end
 
 # Cho Required
-to_field 'id', extract: extract_srw('dc:identifier'), transform: transform(strip: true)
-to_field 'cho_title', extract: extract_srw('dc:title'), transform: transform(strip: true)
+to_field 'id', extract_srw('dc:identifier'), strip
+to_field 'cho_title', extract_srw('dc:title'), strip
 
 # Required per data agreement
 to_field 'cho_provenance', literal('This document is part of BnF website \'Bibliothèques d\'Orient\' - http://heritage.bnf.fr/bibliothequesorient/')
 
 # Cho Other
-to_field 'cho_date', extract: extract_srw('dc:date'), transform: transform(strip: true)
-to_field 'cho_description', extract: extract_srw('dc:description'), transform: transform(strip: true)
-to_field 'cho_dc_rights', extract: extract_srw('dc:rights'), transform: transform(strip: true)
-to_field 'cho_format', extract: extract_srw('dc:format'), transform: transform(strip: true)
-to_field 'cho_publisher', extract: extract_srw('dc:publisher'), transform: transform(strip: true)
-to_field 'cho_relation', extract: extract_srw('dc:relation'), transform: transform(strip: true)
-to_field 'cho_source', extract: extract_srw('dc:source'), transform: transform(strip: true)
-to_field 'cho_subject', extract: extract_srw('dc:subject'), transform: transform(strip: true)
+to_field 'cho_date', extract_srw('dc:date'), strip
+to_field 'cho_description', extract_srw('dc:description'), strip
+to_field 'cho_dc_rights', extract_srw('dc:rights'), strip
+to_field 'cho_format', extract_srw('dc:format'), strip
+to_field 'cho_publisher', extract_srw('dc:publisher'), strip
+to_field 'cho_relation', extract_srw('dc:relation'), strip
+to_field 'cho_source', extract_srw('dc:source'), strip
+to_field 'cho_subject', extract_srw('dc:subject'), strip
 
 # Agg
 to_field 'agg_provider', provider

--- a/bnf_salt_config.rb
+++ b/bnf_salt_config.rb
@@ -1,10 +1,6 @@
 # frozen_string_literal: true
 
-to_field 'cho_contributor', extract: extract_srw('dc:contributor'),
-                            transform: transform(strip: true)
-to_field 'cho_creator', extract: extract_srw('dc:creator'),
-                        transform: transform(strip: true)
-to_field 'cho_edm_type',  extract: extract_srw('dc:type'),
-                          transform: transform(strip: true, prepend: 'text |', split: '|', default: 'text')
-to_field 'cho_language',  extract: extract_srw('dc:language'),
-                          transform: transform(strip: true, gsub: [/Ottoman Turkish/, 'Turkish, Ottoman'])
+to_field 'cho_contributor', extract_srw('dc:contributor'), strip
+to_field 'cho_creator', extract_srw('dc:creator'), strip
+to_field 'cho_edm_type',  extract_srw('dc:type'), strip, prepend('text |'), split('|'), default('text')
+to_field 'cho_language',  extract_srw('dc:language'), strip, gsub(/Ottoman Turkish/, 'Turkish, Ottoman')

